frontend/components/Header.tsx

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import { Grid, Paper, TextField, Typography } from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
 
-export default function Header (props) {
+interface HeaderProps {
```

Comment:
I'm not opposed to switching to an `AppBar` implementation, it would need some customization. I'm not sure about the profile icon, we're not yet sending user info to the FE. I view this as out of scope for this issue, right?

Reply:
I created a separate issue, #182.